### PR TITLE
Add a parameter to allow customising radial track labels

### PR DESCRIPTION
**Pull request by LudoPL**
This is a PR to implement this feature : https://github.com/GeekyAnts/GaugesFlutter/issues/245
Don't hesitate to modify it as you want.

### CODE REVIEW

File: lib/src/radial_gauge/radial_track.dart [hunk 1]
```diff
@@ -121,6 +122,11 @@ class RadialTrack {
   /// The [trackStyle] property is used to customize the track of the Radial Gauge.
   ///
   final TrackStyle trackStyle;
+
+  ///
+  /// The [mapTrackLabel] property is used to customize the track labels of the Radial Gauge.
```
Comment (hasnentai) [inline lib/src/radial_gauge/radial_track.dart:127]: Can we add and example code here ?

**Review by hasnentai — CHANGES_REQUESTED** [review]
Can you also please a screen shot or something which can demonstrate this feature ?

**Comment by LudoPL** [comment]
> Can you also please add screen shot or something which can demonstrate this feature ?

image

File: lib/src/radial_gauge/radial_track.dart (hunk 1 above)
Comment (LudoPL) [reply lib/src/radial_gauge/radial_track.dart:127]: Thanks for your reactivity. I add some example here, in the example directory and a screenshot in the README.

**Comment by LudoPL** [comment]
Sorry, I will close this pull request because I have done it from the "main" branch of my repo.
I will reopen it with a different source branch.

**Pull request closed without merging**